azure-pipelines.yml

```diff
@@ -114,7 +114,7 @@ stages:
             name: $(PoolProvider) # This is a queue-time parameter; Public default is NetCorePublic-Pool; Internal default is NetCoreInternal-Pool
             demands: ImageOverride -equals Build.Ubuntu.1604.Amd64.Open
           ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-            name: Hosted Ubuntu 1604
+            vmImage: ubuntu-latest
```

Comment:
Can probably update the Build.Ubuntu.1604.Amd64.Open above this too.

Reply:
Sure. the problem with staying in a fixed hosted version is this will happen again. The builds that fail with this error are not actually "failing" but are cancelled. So all our automation fails to detect it. I'd rather we break if "latest" becomes incompatible.

Reply:
Sounds good!. I'll update PRs to use the 1804 image and keep official in latest across branches.